Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,260 @@
+        $item[star chart],
+        $effect[none],
+        "",
+        !__quest_state["Level 13"].state_boolean["Richard's star key used"] &&
+            $item[Richard's star key].available_amount() < 1 &&
+            $item[star chart].available_amount() < 1,
+        locationAvailable($location[The Hole In The Sky])
+    ),
+    new Wishable(
+        $item[none],
+        $effect[Frosty],
+        "item/meat for 8-bit",
+        !__quest_state["Level 13"].state_boolean["digital key used"] &&
+            $item[digital key].available_amount() < 1 &&
+            get_property("8BitScore") < 10000,
+        true
+    ),
+    new Wishable(
+        $item[lowercase N],
+        $effect[none],
+        "",
+        !__quest_state["Level 13"].state_boolean["king waiting to be freed"] &&
+            $item[wand of nagamar].available_amount() < 1 &&
+            $item[lowercase N].available_amount() < 1,
+        locationAvailable($location[The Valley of Rof L'm Fao])
+    )
+};
+
+Wishable [int] aftercoreWishables = {
+    new Wishable(
+        $item[bag of foreign bribes],
+        $effect[none],
+        "",
+        locationAvailable($location[The Ice Hotel]),
+        true
+    )
+};
+
+string showWishable(Wishable wishable) {
+    string color = wishable.currentlyAccessible ? "black" : "gray";
+    string wishableStr;
+    string additionalDescription = wishable.additionalDescription != "" ?
+        `: {wishable.additionalDescription}` :
+        "";
+    if (wishable.theItem != $item[none]) {
+        wishableStr = `{wishable.theItem.name}{additionalDescription}`;
+    } else if (wishable.theEffect != $effect[none]) {
+        wishableStr = `{wishable.theEffect.name}{additionalDescription}`;
+    } else {
+        wishableStr = "Unknown item/effect. Report to TourGuide devs >:(";
+    }
+    return HTMLGenerateSpanFont(wishableStr, color);
+}
+
+string [int] showWishables(Wishable [int] wishables) {
+    string [int] currentWishables = {};
+    string [int] futureWishables = {};
+    // My kingdom for polymorphic filter :|
+    foreach index, wishable in wishables {
+        if (!wishable.shouldDisplay) continue;
+
+        if (wishable.currentlyAccessible) {
+            currentWishables.listAppend(showWishable(wishable));
+        } else {
+            futureWishables.listAppend(showWishable(wishable));
+        }
+    }
+    string [int] allWishables = {};
+    allWishables.listAppendList(currentWishables);
+    allWishables.listAppendList(futureWishables);
+    return allWishables;
+}
+
+record MonkeySkill {
+    int fingerCount;
+    skill theSkill;
+    string description;
+};
+
+MonkeySkill [int] monkeySkills = {
+    new MonkeySkill(5, $skill[Monkey Slap], "killbanish"),
+    new MonkeySkill(4, $skill[Monkey Tickle], "delevel"),
+    new MonkeySkill(3, $skill[Evil Monkey Eye], "spooky delevel"),
+    new MonkeySkill(2, $skill[Monkey Peace Sign], "heal"),
+    new MonkeySkill(1, $skill[Monkey Point], "Olfaction-lite")
```

Comment:
Is having all of these actually useful? We could collapse it down to just the 5/1 skills, or even put the whole thing in a table in a tooltip.